symbolic-debuginfo/src/macho.rs

```diff
@@ -252,8 +252,13 @@ impl<'d> MachObject<'d> {
     ///
     /// This is an indication that BCSymbolMaps are needed to symbolicate crash reports correctly.
     pub fn requires_symbolmap(&self) -> bool {
-        self.symbols()
-            .any(|s| s.name().map_or(false, |n| n.starts_with("__?hidden#")))
+        self.symbols().any(|s| {
+            s.name().map_or(false, |n| {
+                n.starts_with("__?hidden#")
+                    || n.starts_with("__hidden#")
+                    || n.starts_with("_hidden#")
```

Comment:
wouldn't it be better to be future proof and regex it?
`hidden#` looks like the common pattern, or even taking into consideration 1 or N `_` and having or not `?`

Reply:
Regexes are a bit expensive at this part, but we could use aho corasick in place of this.

That said, MachO dictates that there can only be a single or two underscores, and I'm not even sure why the symbols I'm looking at don't have the question mark.

Reply:
wfm then, thx for explaining